pjmedia/src/pjmedia/vid_port.c

```diff
@@ -351,17 +351,24 @@ static struct fmt_prop find_closest_fmt(pj_uint32_t req_fmt_id,
 
 	/* Fill the nearest width list. */
 	if (diff_width1 <= diff_width2) {
-	    int k = 1;
-	    pjmedia_rect_size tmp_size = vfd->size;
+	    int k = 0;
+	    while (((k < PJ_ARRAY_SIZE(nearest_width)) &&
+	    (!((vfd->size.w == nearest_width[k].w) && (vfd->size.h == nearest_width[k].h))))) {
```

Comment:
Please modify line formatting, we use 80 characters code width. This includes the code on L366